exporter/datadogexporter/internal/logs/translator.go

```diff
@@ -155,6 +173,18 @@ func extractHostNameAndServiceName(resourceAttrs pcommon.Map, logAttrs pcommon.M
 	return host, service
 }
 
+func decodeTraceID(traceID string) ([16]byte, error) {
+	var ret [16]byte
```

Comment:
What would happen if the traceID is less than 16 bytes long ? can we add test for same.